Riot/Modules/Room/RoomInfo/RoomInfoCoordinator.swift

```diff
@@ -176,9 +176,13 @@ final class RoomInfoCoordinator: NSObject, RoomInfoCoordinatorType {
             coordinator.start()
             push(coordinator: coordinator)
         case .pollHistory:
-            let coordinator: PollHistoryCoordinator = .init(parameters: .init(mode: .active, room: self.room))
+            let coordinator: PollHistoryCoordinator = .init(parameters: .init(mode: .active, room: room, navigationRouter: navigationRouter))
             coordinator.start()
             push(coordinator: coordinator)
+            coordinator.completion = { [weak self] event in
+                guard let self else { return }
+                self.delegate?.roomInfoCoordinator(self, viewEventInTimeline: event)
+            }
```

Comment:
**Optional**
Moving the `push(coordinator: coordinator)` increases readability to me. :-) 

```suggestion
            let coordinator: PollHistoryCoordinator = .init(parameters: .init(mode: .active, room: room, navigationRouter: navigationRouter))
            coordinator.start()
            coordinator.completion = { [weak self] event in
                guard let self else { return }
                self.delegate?.roomInfoCoordinator(self, viewEventInTimeline: event)
            }
            push(coordinator: coordinator)
```